pmd-cataloger/src/main/java/sfdc/sfdx/scanner/messaging/MessageType.java

```diff
@@ -0,0 +1,7 @@
+package sfdc.sfdx.scanner.messaging;
+
+public enum MessageType {
```

Comment:
Why does this need to be in its own file? And it looks like it's not used anywhere outside of the `messaging` package, so does it need to be `public`? Can't it just be default-scoped?

Reply:
Ah, nice one. Thanks for the reminder - I had to move them to separate files back when `EventKey.java` was in a different package. Now that EventKey is also in `messaging`, I can restore this and MessageHandler to original location.